index.js

```diff
@@ -44,16 +46,22 @@ module.exports = {
           replacement: "import lodash from 'lodash'\nconst isPlainObject = lodash.isPlainObject"
         }
       ]
-    })
+    });
+
+    reduxTree = esTranspiler(reduxTree, {
```

Comment:
@rwjblue next problem I found using this approach instead of the broccoli-babel-transpiler like I have it above ^^

The output I get w/ your approach looks like this

```js

;define('redux/index', [], function () {
  'use strict';

  define('index', ['exports', 'createStore', 'combineReducers', 'bindActionCreators', 'applyMiddleware', 'compose', 'utils/warning'], function (exports, _createStore, _combineReducers, _bindActionCreators, _applyMiddleware, _compose, _warning) {
    'use strict';
    Object.defineProperty(exports, "__esModule", {
      value: true
    });
    exports.compose = exports.applyMiddleware = exports.bindActionCreators = exports.combineReducers = exports.createStore = undefined;
    function isCrushed() {}
   // other code
});
});
```

But the correct output (what I get now/ in the app before babel 6)

```js
;define('redux/index', ['exports', 'redux/createStore', 'redux/combineReducers', 'redux/bindActionCreators', 'redux/applyMiddleware', 'redux/compose', 'redux/utils/warning'], function (exports, _reduxCreateStore, _reduxCombineReducers, _reduxBindActionCreators, _reduxApplyMiddleware, _reduxCompose, _reduxUtilsWarning) {

  function isCrushed() {}
  //other code
});
```